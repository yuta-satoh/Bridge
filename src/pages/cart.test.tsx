import { render } from "@testing-library/react";

import Cart from "./cart";
import Cookies from 'js-cookie';

describe('カートページ', () => {
    test('正常に表示', () => {
        Cookies.get = jest.fn().mockImplementation(() => '1');
        const { asFragment } = render(<Cart />);
        expect(asFragment()).toMatchSnapshot();
    })
})
