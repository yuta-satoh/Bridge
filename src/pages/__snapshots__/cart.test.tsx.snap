// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`カートページ 正常に表示（ゲストカートの中身あり） 1`] = `
<DocumentFragment>
  <div
    class="w-4/5 mx-auto"
  >
    <h2
      class="text-3xl font-bold mt-10"
    >
      カート
    </h2>
    <div
      class="block py-40 mx-auto my-10 bg-neutral-100 rounded"
    >
      <h2
        class="text-center text-3xl font-bold mb-10"
      >
        カートに商品がありません
      </h2>
      <a
        href="/"
      >
        <div
          class="container mx-auto pt-1.5 text-center h-10 w-1/3 border-2 border-neutral-900 bg-white mt-4"
        >
          お買い物はこちらから
        </div>
      </a>
    </div>
  </div>
</DocumentFragment>
`;

exports[`カートページ 正常に表示（空のゲストカート） 1`] = `
<DocumentFragment>
  <div
    class="w-4/5 mx-auto"
  >
    <h2
      class="text-3xl font-bold mt-10"
    >
      カート
    </h2>
    <div
      class="block py-40 mx-auto my-10 bg-neutral-100 rounded"
    >
      <h2
        class="text-center text-3xl font-bold mb-10"
      >
        カートに商品がありません
      </h2>
      <a
        href="/"
      >
        <div
          class="container mx-auto pt-1.5 text-center h-10 w-1/3 border-2 border-neutral-900 bg-white mt-4"
        >
          お買い物はこちらから
        </div>
      </a>
    </div>
  </div>
</DocumentFragment>
`;

exports[`カートページ 正常に表示（空のユーザーカート） 1`] = `
<DocumentFragment>
  <div
    class="w-4/5 mx-auto"
  >
    <h2
      class="text-3xl font-bold mt-10"
    >
      カート
    </h2>
    <div
      class="block py-40 mx-auto my-10 bg-neutral-100 rounded"
    >
      <h2
        class="text-center text-3xl font-bold mb-10"
      >
        カートに商品がありません
      </h2>
      <a
        href="/"
      >
        <div
          class="container mx-auto pt-1.5 text-center h-10 w-1/3 border-2 border-neutral-900 bg-white mt-4"
        >
          お買い物はこちらから
        </div>
      </a>
    </div>
  </div>
</DocumentFragment>
`;
